extractor/src/main/java/org/schabi/newpipe/extractor/stream/AudioStream.java

```diff
@@ -25,6 +25,14 @@
 public class AudioStream extends Stream {
     public int average_bitrate = -1;
 
+    // Fields for Dash
+    public int bitrate;
+    public int initStart;
+    public int initEnd;
+    public int indexStart;
+    public int indexEnd;
+    public String codec;
+
```

Comment:
```suggestion
    // Fields for Dash
    private int bitrate;
    private int initStart;
    private int initEnd;
    private int indexStart;
    private int indexEnd;
    private String codec;

```

you created getters for this attributes below…